rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Looks like this was discussed and then removed in https://issues.jboss.org/browse/WINDUPRULE-135 -- will let Marek take a look, but we should discuss if this needs to be added back.